src/qt/forms/debugwindow.ui

```diff
@@ -249,6 +249,29 @@
         </widget>
        </item>
        <item row="9" column="0">
+        <widget class="QLabel" name="label_14">
+         <property name="text">
+          <string>Local Address</string>
```

Comment:
we could add a translator comment here by adding in the `extracomment` field:

```suggestion
          <string extracomment="nice translator comment">Local Address</string>
```

Reply:
While here, if there's more than one local address; I guess this should take the plural form, no?

Reply:
Or could be just changed to "Local Address(es)" maybe?